Dependency of rocksdbjni cannot support ARM platform

**Alluxio Version:**
What version of Alluxio are you using?

**Describe the bug**
The dependency `rocksdbjni` of current version cannot support ARM platform, see following error:
```
[ERROR] Tests run: 37, Failures: 0, Errors: 28, Skipped: 2, Time elapsed: 1.312 s <<< FAILURE! - in alluxio.client.file.cache.store.PageStoreTest
[ERROR] getSmallBuffer[0](alluxio.client.file.cache.store.PageStoreTest)  Time elapsed: 0.657 s  <<< ERROR!                    
java.lang.UnsatisfiedLinkError: /tmp/librocksdbjni12713547637208397247.so: /tmp/librocksdbjni12713547637208397247.so: cannot open shared object file: No such file or directory (Possible cau
se: can't load AMD 64-bit .so on a AARCH64-bit platform)
```

**To Reproduce**
```shell
alluxio.util.FileSystemOptionsTest
```

**Expected behavior**
Tests of Alluxio can pass on ARM64 platform

**Urgency**
Medium

**Additional context**
N/A

--- a/pom.xml
+++ b/pom.xml
@@ -142,7 +142,7 @@
     <glusterfs-hadoop.version>2.3.13</glusterfs-hadoop.version>
     <grpc.version>1.28.1</grpc.version>
     <netty.version>4.1.51.Final</netty.version>
-    <rocksdb.version>5.15.10</rocksdb.version>
+    <rocksdb.version>5.18.4</rocksdb.version>
     <hadoop.version>3.3.0</hadoop.version>
     <jacoco.version>0.8.5</jacoco.version>
     <java.version>1.8</java.version>

